src/components/RichTextPopup/RichTextPopup.js

```diff
@@ -192,6 +209,12 @@ const RichTextPopup = () => {
             img="ic_annotation_strikeout_black_24px"
             title="option.richText.strikeout"
           />
+          <Button
+            dataElement="richTextShowMathSymbols"
+            onClick={handleClick('showSymbols')}

```

Comment:
It looks like in `handleClick` we now have a special check for `showSymbols`. I prefer having a separate function for this and rename the original `handleClick` to something like `handleFormatChange` to remove the special check.

This is not a blocker but more like a nitpick, so no need to make a change if my suggestion isn't liked.

Reply:
I agree. I rename the `handleClick` function to `handleTextFormatChange` and add a new function called `handleSymbolsClick`